### ISSUE TITLE
Autocomplete for user-defined functions in Postgres

### ISSUE THREAD

**Issue by fvannee** [dbeaver/dbeaver#20261]
### Description

When turning on autocomplete for functions, this only enables it for the functions in the column list. However, often functions are used in the relation-part of the query, but this does not show up in DBeaver. Is there another option to show auto-complete for functions here?

### DBeaver Version

23.1

### Operating System

Windows

### Database and driver

Postgresql 15 / JDBC

### Steps to reproduce

```
CREATE OR REPLACE FUNCTION public.test_f(_val int)
 RETURNS table (a int)
 LANGUAGE sql
 IMMUTABLE PARALLEL SAFE
AS $function$
   SELECT * FROM generate_series(1, _val)
$function$
;

-- here dbeaver does not show autocomplete for test_f
select * from public.test_f(10);
```

**Comment by E1izabeth** [comment]
When I write the scheme name first, autocompletion for procedures do not appears.
It appears for me when I try without scheme, but only after I write `test_`
![image](https://github.com/dbeaver/dbeaver/assets/28875055/69f03ec0-62e0-4d6e-b65f-b277915e811f)

**Comment by E1izabeth** [comment]
Probably, we should improve that. Thank you for bug report.

**Comment by fvannee** [comment]
You're right - omitting `public.` does seem give the autocomplete suggestions after writing the underscore. This only works if the function is created in the `public` schema though. If the function exists in a different schema it needs to be prefixed and then autocomplete doesn't work.

Thanks for looking into it!

**Comment by pyrocks** [comment]
Any update on this? I use schemas heavily and I always use them in the `FROM` clause.
Thanks
